index.mjs

```diff
@@ -18,12 +18,16 @@ export default function viteSquint(opts = {}) {
         return { code: compiled, map: null };
       }
     },
-    resolveId(id, imported, options) {
+    resolveId(id, importer, options) {
+      if (/\.cljs.jsx$/.test(id)) {
+        // other plugins might make us resolve ourselves
+        return path.resolve(dirname(importer), id);
+      }
```

Comment:
Can you expand on this? Like I'm not sure I get the comment 😇

Reply:
I expanded on this. Please let me know if this is more clear

Reply:
Thanks that makes a lot of sense!